insonmnia/worker/overseer.go

```diff
@@ -603,6 +607,28 @@ func (o *overseer) Stop(ctx context.Context, containerid string) error {
 	return descriptor.Kill(ctx)
 }
 
+func (o *overseer) PurgeContainer(ctx context.Context, containerID string) error {
```

Comment:
what's the difference of this function and OnDealFinish?

Reply:
`OnDealFinish` assumes that worker has all possible info about the running container and will abort if some of this info is missing (which is not what we want).